docs/networking.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Networking
+subtitle: Connecting ev3dev to the internet and other devices
+---
+
+If you want to access your ev3dev installation from a PC or install new software
+on your EV3, you'll need to connect to a network. Here are some resources to
+help you out.
+
+# Connecting to the internet
+
+You have a few options for connecting to the internet.
+
+### With a Wi-Fi dongle
+
+The simplest route is to purchase a Wi-Fi dongle and plug it into the EV3's USB
```

Comment:
I would second @WasabiFan here.

Whatever you do you will likely endup using Wi-Fi.  Or just most people will.

Other means of connection are great if one can't use Wi-Fi for some reason (USB host used by something else, no dongle yet, no driver for dongle, etc).